test/api_status_app_version_test.dart

```diff
@@ -1,9 +1,9 @@
-import 'package:openapi/api.dart';
+import 'package:tba_api_client/api.dart';
```

Comment:
I feel like these namespace changes should be handled by the generator....

Reply:
They should, and yet here we are

Reply:
haha fair enough